ChocolArm64/Optimizations.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.Intrinsics.X86;
+
+public static class Optimizations
+{
+    internal static bool FastFP = true;
+
+    private static bool _useAllSseIfAvailable = true;
```

Comment:
These can be turned into `const`s which can then be capitalized as, for example, `UseAllSseIfAvailable`.

```suggestion
    private const bool UseAllSseIfAvailable = true;
```